File: Android/app/src/main/java/uk/ac/lshtm/mantra/android/settings/SettingsActivity.java
package uk.ac.lshtm.mantra.android.settings;

import android.app.Activity;
import android.os.Bundle;

import androidx.annotation.Nullable;

import uk.ac.lshtm.mantra.android.R;

public class SettingsActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
    }
}

File: Android/app/src/main/java/uk/ac/lshtm/mantra/android/settings/SettingsActivity.kt
package uk.ac.lshtm.mantra.android.settings

import android.app.Activity
import android.os.Bundle
import uk.ac.lshtm.mantra.android.R

class SettingsActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)
    }
}